.github/workflows/run-tests.yml

```diff
@@ -59,12 +59,14 @@ jobs:
       with:
         php-version: ${{ matrix.php }}
         coverage: none
+        tools: composer:v2
         
     - name: Install dependencies
       run: |
+        composer self-update --2
```

Comment:
This should be redundant as setup-php does this already.

Reply:
Yeah I added it because of the weird error I got, wanted to make sure it was the very latest version.